gr-utils/python/modtool/gr-newmod/CMakeLists.txt

```diff
@@ -190,11 +190,15 @@ add_custom_target(uninstall
 ########################################################################
 add_subdirectory(include/howto)
 add_subdirectory(lib)
-add_subdirectory(swig)
-add_subdirectory(python)
-add_subdirectory(grc)
 add_subdirectory(apps)
 add_subdirectory(docs)
+if(ENABLE_PYTHON)
```

Comment:
I'd prefer that this change for ENABLE_PYTHON is minimally in a separate commit, if not a separate PR.